Docker/docker-compose.yml

```diff
@@ -11,109 +11,136 @@ services:
             - var-run-postgresql:/var/run/postgresql
     # Once off container to run the DB migrations
     migrate:
-        build: ./nab
+        build:
+            context: ..
+            dockerfile: Docker/nab/Dockerfile
         volumes:
             - ../:/home/pi/pynab
             - var-run-postgresql:/var/run/postgresql
         depends_on:
             - db
         command: ["/bin/bash", "/usr/local/bin/run-migrate.sh"]
-    nabdevd:
-        image: registry.gitlab.com/nguillaumin/nabdevd:latest
+    nabd:
+        build:
+            context: ..
+            dockerfile: Docker/nab/Dockerfile
+        volumes:
+            - ../:/home/pi/pynab
+            - var-run-postgresql:/var/run/postgresql
+        command: ["/bin/sh", "-c", "ln -sf /dev/stdout /var/log/nabd && /home/pi/venv/bin/python3 -m nabd.nabd"]
```

Comment:
I think the symlink should point to `/var/log/nab.log` (Missing `.log`)

Reply:
Thanks!